sote/libsote/inorganic_stone_layer.lua

```diff
@@ -0,0 +1,211 @@
+local isl = {}
```

Comment:
This belongs in raws, not in libsote, imho

Reply:
I've also considered this, in raws we already have bedrocks, but they lack some of the attributes the materials from C# have and need, so I decided not to extend bedrocks raws, nor to put the stone materials together with bedrocks, which might cause confusion.

Reply:
They are the exact same thing, if we are porting world gen they will need this new data

Reply:
Makes sense from that perspective. I will see about merging them.

Reply:
Just an observation, ROCK_SALT, DOLOMITE and OBSIDIAN are missing from bedrocks.

Reply:
I think I removed them as they weren't implemented well enough. We can add them back for the purpose of the port @dsecrieru

Reply:
I can add them, but I don't know what values to input for sand, clay, silt, organics, minerals and weathering, so I'll just default them to 0 for now.
Also note that there is an "obsidian" resource, based on "rhyolite" bedrock. No idea if that has any relevance or not.

Reply:
Looks like "dolomite" is present in bedrocks as "dolostone"

Reply:
Quite possible, it's been a while, I don't remember. Tell me when you think it's ready for merging again @dsecrieru ^-^